modules/Microsoft.DataProtection/backupVaults/readme.md

```diff
@@ -365,55 +365,8 @@ module backupVaults './Microsoft.DataProtection/backupVaults/deploy.bicep' = {
           ]
           objectType: 'BackupPolicy'
           policyRules: [
-            {
-              backupParameters: {
-                backupType: 'Incremental'
-                objectType: 'AzureBackupParams'
-              }
-              dataStore: {
-                dataStoreType: 'OperationalStore'
-                objectType: 'DataStoreInfoBase'
-              }
-              name: 'BackupDaily'
-              objectType: 'AzureBackupRule'
-              trigger: {
-                objectType: 'ScheduleBasedTriggerContext'
-                schedule: {
-                  repeatingTimeIntervals: [
-                    'R/2022-05-31T23:30:00+01:00/P1D'
-                  ]
-                  timeZone: 'W. Europe Standard Time'
-                }
-                taggingCriteria: [
-                  {
-                    isDefault: true
-                    taggingPriority: 99
-                    tagInfo: {
-                      id: 'Default_'
-                      tagName: 'Default'
-                    }
-                  }
-                ]
-              }
-            }
-            {
-              isDefault: true
-              lifecycles: [
-                {
-                  deleteAfter: {
-                    duration: 'P7D'
-                    objectType: 'AbsoluteDeleteOption'
-                  }
-                  sourceDataStore: {
-                    dataStoreType: 'OperationalStore'
-                    objectType: 'DataStoreInfoBase'
-                  }
-                  targetDataStoreCopySettings: []
-                }
-              ]
-              name: 'Default'
-              objectType: 'AzureRetentionRule'
-            }
+            'System.Management.Automation.OrderedHashtable'
```

Comment:
@AlexanderSehr

Reply:
Jap, this is wrong ;)

Reply:
@JPEasier how did you generate this?

Reply:
Once #2315 is merged, please merge latest main into your branch and re-run the `Set-ModuleReadMe` script

Reply:
@JPEasier , you're good to go :)

Reply:
@eriqua, @AlexanderSehr , the readme is updated.